deps/build.jl

```diff
@@ -0,0 +1,15 @@
+if haskey(ENV, "CONDA_JL_HOME")
+    default_dir = ENV["CONDA_JL_HOME"]
+elseif isfile("deps.jl")
+    include("deps.jl")
+else
+    default_dir = abspath(dirname(@__FILE__), "usr")
+end
+
+deps = "default_dir=\"$(escape_string(default_dir))\""
+
+if !isfile("deps.jl") || readchomp("deps.jl") != deps
+    open("deps.jl", "w") do f
```

Comment:
You can just do `write("deps.jl", deps)` rather than `open ... println ... end`, in which case you would include the newline in `deps` and use `readstring` rather than `readchomp`.